akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala

```diff
@@ -1169,9 +1169,9 @@ abstract class StressSpec
       enterBarrier("after-" + step)
     }
 
-    // FIXME issue #21810
+    // Aeron UDP with embedded driver seems too heavy to get to pass
```

Comment:
for the record: we had a shared media driver here previously but it was removed in https://github.com/akka/akka/pull/27037

anyway, let's not spend time on that